lib/lndclient/LndClient.ts

```diff
@@ -72,6 +73,25 @@ class LndClient extends BaseClient {
     });
   }
 
+  private connect = (lndCLient: LndClient, subscribeInvoices: boolean = false) => {
+    const { host, port } = lndCLient.config;
+
+    lndCLient.lightning = new LightningClient(`${host}:${port}`, lndCLient.credentials);
+
+    lndCLient.getInfo().then((getInfoResponse) => {
```

Comment:
We are trying to use the async/await pattern everywhere in the codebase, so make this an `async` function and do `const getInfoResponse = await this.getInfo();` inside a try/catch statement.